api/src/controllers/admin/deleteRole.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from 'express';
+import { CamelCasedProperties } from 'type-fest';
+import type * as s from 'zapatos/schema';
+
+import * as auth0Repository from '../../repositories/auth0Repository';
+import * as userRolesRepository from '../../repositories/userRolesRepository';
+import Role from '../../types/roles';
+import { decodeQueryToUser, manyToCamelCase } from '../../util/helper';
+import controller from '../controllerUtil';
+import Validator, { beAValidRole, beAValidUser } from '../validation';
+
+type ResBody = {
+    users?: CamelCasedProperties<s.user_roles.JSONSelectable>[];
+    message?: string;
+};
+
+type Params = {
+    userId: string;
+    role: string;
+};
```

Comment:
`Params` are usually used for route parameters, e.g. `users/{userId}/roles/{role}` did you mean to use `ReqBody`?